docs/js-core-1/week-1/lesson.md

```diff
@@ -251,80 +345,143 @@ function double(number) {
+
+
+---
+<br />
+
+2. Create a function called `increaseByHalf` that should
+
+- take a number as an input
+- return this input number **increased by a half**
+
+E.g. `increaseByHalf(50)` should **evaluate** to `75` 
+
+E.g. `increaseByHalf(100)` should **evaluate** to `150`
+
+In your groups, think carefully about the following things:
 
-## Calling functions inside functions
+- what the parameter name should be
+- creating good names for any variables you use
+- don't use `console.log` inside the body of the function
 
-Functions are very powerful.
+Call `increaseByHalf` a few times with different inputs and `console.log` the outputs in order to check your function is working properly.
 
-- You can write more than one line of code inside of functions.
-- You can use variables inside of functions.
-- You can call other functions inside of functions!
+
+Now declare a variable called `pocketMoney` and assign to it a value of `413`.<br />
+Someone's been generous and decided to increase your `pocketMoney` by 50% 😎
+
+Using the functions `increaseByHalf` and `formatPenceToPounds` only, find the new amount of pocket money as a string in pounds with a `£` symbol at the front.
+
+
+---
+<br />
+
+3. Check out the code below:
 
 ```js
-function getAgeInDays(age) {
-  return age * 365;
+function printMessage(x) {
+  console.log('Hello, nice to meet you ' + x);
 }
 
-function createGreeting(name, age) {
-  const ageInDays = getAgeInDays(age);
-  const message =
-    "My Name is " + name + " and I was born over " + ageInDays + " days ago!";
-  return message;
+function getMessage(name) {
+  return 'Hello, nice to meet you ' + name;
 }
+
+printMessage('Mitch');
+
+printMessage('August');
+
+getMessage('Laetitia');
 ```
 
-### Exercise (20 mins)
+The functions above behave similarly but _differ_ in some important ways.
+
+In groups, study the code above and then answer the following questions:
+
+a) How many times is the function `printMessage` called ?<br />
+b) How many times is the function `getMessage` called ?<br />
+c) What is the parameter name for the function `printMessage` ?<br />
+d) What is the parameter name for the function `getMessage` ?<br />
+
+Discuss briefly which **parameter name** is better out of the two functions and why ?
+
+e) What is the **return value** of the function `printMessage` whenever it is called ?
 
-1. Write a function that returns the year someone is born given their age as input
-2. Using the answer from step 1, write a function that takes someone's name and age as input and returns a string that states the person's name and year they were born in a sentence
+
+Someone then creates 2 further variables `messageForAli` and `curiousMessage`:
+
+```js
+const messageForAli = getMessage('Ali');
+
+const curiousMessage = getMessage(42);
+```
+
+f) What will the variable `messageForAli` evaluate to when the code above is executed ?<br />
+g) What will the variable `curiousMessage` evaluate to when the code above is executed ?
```

Comment:
I wonder if it's worth adding a point about how to find out what the answers are?

I understand that one of the goals is to reduce confusion about `console.log`, but I think this is a good opportunity to show that it should be used for debugging purposes.